### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by rgoomar**
Helps with https://github.com/nanobox-io/nanobox/issues/503

### CODE REVIEW

File: commands/info.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ var (
 
 	// InfoCmd ...
 	InfoCmd = &cobra.Command{
-		Use:   "info",
+		Use:   "info [local | dry-run | remote-alias]",
```
Comment (glinton) [inline commands/info.go]: I believe the `remote-alias` portion of this has yet to be implemented

File: commands/login.go [hunk 2]
```diff
@@ -15,7 +15,11 @@ var (
 	LoginCmd = &cobra.Command{
 		Use:   "login",
 		Short: "Authenticate your nanobox client with your nanobox.io account.",
-		Long:  ``,
+		Long:  `
+		Authenticate with your nanobox account by passing the username and password in or using the following environment variables:
```
Comment (glinton) [inline commands/login.go]: I suggest removing the leading spaces on these 3 lines, so the output isn't pushed out so much.

File: commands/remote.go [hunk 3]
```diff
@@ -27,7 +27,7 @@ remote, live applications created with Nanobox.
 
 	// RemoteAddCmd ...
 	RemoteAddCmd = &cobra.Command{
-		Use:   "add",
+		Use:   "add <team-name>/<app-name> [remote-alias]",
```
Comment (glinton) [inline commands/remote.go]: Since `team-name` is technically optional, can you update this to be `[team-name/]<app-name> [remote-alias]`

**Comment by rgoomar** [comment]
@glinton should be good to go now

**Pull request merged**